openmetadata-service/src/main/java/org/openmetadata/service/elasticsearch/ElasticSearchEventPublisher.java

```diff
@@ -93,7 +93,7 @@
 import org.openmetadata.service.events.errors.EventPublisherException;
 import org.openmetadata.service.jdbi3.CollectionDAO;
 import org.openmetadata.service.resources.elasticsearch.BuildSearchIndexResource;
-import org.openmetadata.service.resources.events.EventResource.ChangeEventList;
+import org.openmetadata.service.resources.events.ChangeEventResource.ChangeEventList;
 import org.openmetadata.service.util.ElasticSearchClientUtils;
```

Comment:
Let's just call this EventResource instead of ChangeEventResource. Also EventList instead of ChangeEventList.

Reply:
done